src/main/java/org/damap/base/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -319,6 +319,19 @@ public void datasetsInformation() {
     if (dmp.getTargetAudience() != null)
       addReplacement(replacements, "[targetaudience]", dmp.getTargetAudience());
     else addReplacement(replacements, "[targetaudience]", "");
+
+    StringBuilder reusedDescription = new StringBuilder();
+    List<Dataset> reusedDatasets = getReusedDatasets();
+    if (!reusedDatasets.isEmpty()) {
```

Comment:
The conditional check can be removed, as the for loop will have no effect if the datasets are empty